include/boost/multiprecision/rational_adaptor.hpp

```diff
@@ -1,114 +1,83 @@
 ///////////////////////////////////////////////////////////////
-//  Copyright 2011 John Maddock. Distributed under the Boost
+//  Copyright 2020 John Maddock. Distributed under the Boost
 //  Software License, Version 1.0. (See accompanying file
 //  LICENSE_1_0.txt or copy at https://www.boost.org/LICENSE_1_0.txt
 
-#ifndef BOOST_MATH_RATIONAL_ADAPTER_HPP
-#define BOOST_MATH_RATIONAL_ADAPTER_HPP
+#ifndef BOOST_MATH_SKELETON_BACKEND_HPP
+#define BOOST_MATH_SKELETON_BACKEND_HPP
 
-#include <iostream>
-#include <iomanip>
-#include <sstream>
-#include <cstdint>
-#include <boost/multiprecision/detail/hash.hpp>
 #include <boost/multiprecision/number.hpp>
-#ifdef BOOST_MSVC
-#pragma warning(push)
-#pragma warning(disable : 4512 4127)
-#endif
-#include <boost/rational.hpp>
-#ifdef BOOST_MSVC
-#pragma warning(pop)
-#endif
+#include <boost/container_hash/hash.hpp>
```

Comment:
So this is just one step along the way to a standalone?

Reply:
Baby steps yes, but as it happens that #include can be removed now :)